tests/ethereum/test_general.py

```diff
@@ -62,6 +69,33 @@ def test_int_ovf(self):
         self.assertIn("Unsigned integer overflow at MUL instruction", all_findings)
 
 
+class EthVerifierIntegrationTest(unittest.TestCase):
+    def test_propverif(self):
+        smtcfg = config.get_group("smt")
+        smtcfg.solver = smtcfg.solver.yices
+        with smtcfg.temp_vals():
+            smtcfg.solver = smtcfg.solver.yices
+
+            filename = os.path.join(THIS_DIR, "contracts/prop_verifier.sol")
+            f = io.StringIO()
+            with contextlib.redirect_stdout(f):
+                verifier.manticore_verifier(filename, "TestToken")
+            output = f.getvalue()
+            self.assertIsNotNone(
+                re.compile(
+                    r".*crytic_test_balance\s*\|\s*failed\s*\([0-9a-f]+\).*", re.DOTALL
```

Comment:
Weirdly enough, [CodeCov is saying that the line of code that prints out this test case is not covered](https://codecov.io/gh/trailofbits/manticore/compare/61909a157bd621b273bd5eabd685aef4d7c26af9...8c80a1d984f9f6ecd8887dc5cb7f9764d8938845/diff#D1-332
). However, running the verifier on this contract myself, the output looks fine. :shrug:

The upload seems to have succeeded, so I don't know what to make of it.